pytest_catchlog.py

```diff
@@ -92,19 +97,81 @@ def pytest_addoption(parser):
                     help='disable printing caught logs on failed tests.')
     add_option_ini(parser,
                    '--log-format',
-                   dest='log_format', default=('%(filename)-25s %(lineno)4d'
-                                               ' %(levelname)-8s %(message)s'),
+                   dest='log_format',
+                   default=DEFAULT_LOG_FORMAT,
                    help='log format as used by the logging module.')
     add_option_ini(parser,
                    '--log-date-format',
-                   dest='log_date_format', default=None,
+                   dest='log_date_format',
+                   default=DEFAULT_DATE_FORMAT,
                    help='log date format as used by the logging module.')
+    add_option_ini(parser,
+                   '--log-extra-level',
+                   dest='log_extra_levels',
```

Comment:
What's about renaming it to `--log-level-extra` and 'log_level_extra' respectively? These would look more consistent with `--log-level` that I plan to add later. @The-Compiler What would you say?